components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -27,20 +27,22 @@ type Props = Pick<AIChatContext,
   | 'isToolsMenuOpen'
   | 'setIsToolsMenuOpen'
   | 'isMobile'
-  | 'hasInitialHistory'
   | 'shouldDisableUserInput'
   | 'hasAcceptedAgreement'>
 
 interface InputBoxProps {
     context: Props
     onFocusInputMobile?: () => unknown
+    maybeShowSoftKeyboard?: (querySubmitted: boolean) => unknown
 }
 
 function InputBox(props: InputBoxProps) {
   const onInputChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
     props.context.setInputText(e.target.value)
   }
 
+  const querySubmitted = React.useRef(false)
```

Comment:
Optional: I'm pretty sure we could use `useState` here instead of useRef (though not 100%).